src/_data/fulltime_curriculum_fr.json

```diff
@@ -0,0 +1,95 @@
+{
+  "units": [
+    {
+      "name": "Programmation",
+      "content": [
+        {
+          "type": "paragraph",
+          "text": "Dans cette section, nous allons apprendre les bases de la programmation en construisant diverses applications terminales. Vous aurez la chance de construire une application météo de ligne de commande, un carnet d'adresses ainsi que quelques jeux de chance."
+        },
+        {
+          "type": "list",
+          "items": [
+            "Mise en place de notre IDE et environnement de développement",
+            "Apprendre les bases de la ligne de commande",
+            "Utilisation de Git et GitHub",
+            "JavaScript et JSON",
+            "Introduction a NodeJS et npm",
+            "Caractéristiques de ECMAScript 6",
+            "Object-Oriented Programming"
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Bases de données et développement Back-End",
+      "content": [
+        {
+          "type": "paragraph",
+          "text": "Dans cette section, nous allons nous concentrer sur la modélisation des données et de la persistance avec MySQL. Nous allons aussi pratiquer exposer les données en créant une projets API qui comprennent la mise à niveau de notre carnet d'adresses et jeux à utiliser une base de données."
+        },
+        {
+          "type": "list",
+          "items": [
+            "Présentation des options de persistance des données",
+            "Discussion sur SQL vs NoSQL",
```

Comment:
@ziad-saab Suggesting removal of this topic since we excluded from original course?